drivers/hmis/spec/models/hmis/form/form_processor_editor_user_ids_spec.rb

```diff
@@ -0,0 +1,80 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
```

Comment:
This would've gone in `form_processor_spec.rb` but that file is getting so enormous, so I'm proposing to separate it out here.